webapp/src/components/backstage/playbook_runs/playbook_run/status_update.tsx

```diff
@@ -166,7 +168,11 @@ export const ViewerStatusUpdate = ({id, playbookRun, openRHS, lastStatusUpdate}:
                         {dueInfo.time}
                     </DueDateViewer>
                     {playbookRun.current_status === PlaybookRunStatus.InProgress ? (
-                        <RequestUpdateButton onClick={showRequestUpdateConfirm}/>
+                        <RequestUpdateButton
+                            onClick={showRequestUpdateConfirm}
+                            type={'button'}
+                            disabled={false}
```

Comment:
Should we disable the button when the run is finished?
```suggestion
                            disabled={playbookRun.current_status === PlaybookRunStatus.Finished}
```